storage/blob.go

```diff
@@ -439,7 +439,9 @@ func (b BlobStorageClient) ListContainers(params ListContainersParameters) (Cont
 	if err != nil {
 		return out, err
 	}
-	defer resp.body.Close()
+	defer func() {
```

Comment:
@mcardosos I strongly suggest turning off `errcheck` for this package. It makes the code really ugly with this change. It's idiomatic go to call `defer resp.body.Close()` and discard the output implicitly.

I am sorry that I did not see this coming. I thought you were just going to change 1-2 places in `management` package. But this results in quite ugly code for this package.

We must use static analysis tools like this occasionally to catch bugs but for cases like these, they are deliberate decisions and we should not be using hacks like this all over the place to suppress the static analysis tool. I think `errcheck` is a bit extreme. Perhaps just run `errcheck | grep -v Close` or something again. :(